modules/addons/src/gltf/gltf-animator.js

```diff
@@ -64,6 +64,38 @@ function applyTranslationRotationScale(gltfNode, node) {
   }
 }
 
+function linearInterpolate(path, t, a, b) {
+  // TODO: implement SLERP when path is rotation
+  return a.map((v, index) => v * (1 - t) + b[index] * t);
+}
+
+function interpolate(time, {input, interpolation, output}, path) {
+  const maxTime = input[input.length - 1];
+  const animationTime = time % maxTime;
+
+  const nextIndex = input.findIndex(t => t >= animationTime);
+  const previousIndex = Math.max(0, nextIndex - 1);
+
+  switch (interpolation) {
+    case 'STEP':
```

Comment:
This is the same as `default`, so you can just remove this and put a comment under `default` that it's considered `STEP`.

Reply:
Default is not STEP. Fixed it now to print warning.